packages/flutter/lib/src/widgets/form.dart

```diff
@@ -635,4 +662,13 @@ enum AutovalidateMode {
   /// Used to auto-validate [Form] and [FormField] only after each user
   /// interaction.
   onUserInteraction,
+
+  /// Used to auto-validate current [FormField] when unfocused.
+  ///
+  /// This mode is useful when you want to validate a [FormField] after the user
+  /// has interacted with another one.
```

Comment:
If the user types into one field and then taps the background to remove focus, it still validates, right?

Reply:
Yes he will go.